nautobot/dcim/models/power.py

```diff
@@ -137,31 +174,119 @@ class PowerFeed(PrimaryModel, PathEndpoint, CableTermination):
         "voltage",
         "amperage",
         "max_utilization",
+        "breaker_position",
+        "breaker_pole_count",
         "available_power",
     ]
 
     class Meta:
         ordering = ["power_panel", "name"]
```

Comment:
Occurs to me to wonder whether we should change this to something like `["power_panel", "breaker_position", "name"]` instead